HHS/OPRE-OPS
# Ops 1174 notification center modal

## What changed

Add styling and notification content to Notification Center.

## Issue

https://github.com/HHS/OPRE-OPS/issues/1174

## How to test

1. Log into OPS and click the bell for your notifications.
2. Verify that the Notification Center is styled according to the current design

## Screenshots

![image](https://github.com/HHS/OPRE-OPS/assets/1972158/15b9002a-c5f1-4265-a873-3d8485bc76bb)
